.github/ISSUE_TEMPLATE/request.yml

```
@@ -0,0 +1,30 @@
+name: "Feature request"
+description: "Create a feature request"
+labels:
+  - "feature"
+body:
```

```suggestion

body:
```

`bug_report.yml` と相違があるのが気になる.